src/ReturnBookControl.java

```diff
@@ -1,77 +1,75 @@
 class ReturnBookControl {
 
-	private ReturnBookUI ui;
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE state;
+	private ReturnBookUi returnBookUi;
+	private enum controlState { INITIALISED, READY, INSPECTING };
+	private controlState state;
 	

```

Comment:
Sorry, still a break on line six that isn't required. - I am being pedantic but I think we need to be?